.github/workflows/generate.yml

```diff
@@ -33,23 +33,27 @@ jobs:
           go-version: '^1.17.3'
 
       - name: Generate CPE database
-        run: cd fleet && go mod download && go run -tags fts5 cmd/cpe/generate.go && cat ./etagenv >> $GITHUB_ENV
+        run: |
```

Comment:
Updated to be more readable. `|` preserves newlines. By default the shell is `/bin/bash -e` which means any errors cause an early exit so no need to `&&` everything together.